src/haz3lschool/SchoolExercise.re

```diff
@@ -588,6 +588,8 @@ module F = (ExerciseEnv: ExerciseEnv) => {
 
   type stitched_statics = stitched(StaticsItem.t);
 
+  /* Generates the UExp terms and corresponding
```

Comment:
Try "Stitching is necessary to concatenate terms from different editors, which are then typechecked. Multiple stitchings are needed for each exercise (see comments in the stitched type above)".